crates/rpc/rpc-eth-types/src/pending_block.rs

```diff
@@ -120,4 +122,6 @@ pub struct PendingBlock {
     pub block: SealedBlockWithSenders,
     /// Timestamp when the pending block is considered outdated
     pub expires_at: Instant,
+    /// The receipts for the pending block
+    pub receipts: Vec<Option<Receipt>>,
```

Comment:
hmm, do we need to store Option here?
ideally this is just a Vec

perhaps this is due to some executor API? if so we should do the flatten + collect when we set this